tests/testJointcal_cfht.py

```diff
@@ -0,0 +1,57 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import unittest
+import os
+
+from lsst.afw import geom, coord
+import lsst.utils
+import lsst.pex.exceptions
+
+import jointcalTestBase
+
+try:
+    data_dir = lsst.utils.getPackageDir('validation_data_jointcal')
+    os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(data_dir, 'cfht_and_index')
+except lsst.pex.exceptions.NotFoundError:
+    data_dir = None
+
+# We don't want the absolute astrometry to become significantly worse
+# than the single-epoch astrometry (about 0.040").
+absolute_error = 48e-3/jointcalTestBase.arcsec_per_radian
```

Comment:
"about 40 mas". Single epoch was around 40 (different per image), and I had to pick something a bit larger than for the test to succeed. I thought the comment was clear on that: how can I make it clearer?